### ISSUE TITLE
Do we need AEROBULK_INIT?

### ISSUE THREAD

**Issue by jbusecke** [xgcm/aerobulk-python#36]
While digging into #31 with @paigem I looked at the aerobulk code, and it seems they do not use the mask for the computation at all? As far as I can see they only time the mask is used in the autodetermination of the humidity and the range checks of inputs [here](https://github.com/brodeau/aerobulk/blob/115d6db4fde9a3c1943a46bf51867abb5ca903b4/src/mod_aerobulk.f90#L126-L153). The actual computation still runs on those values? If the calculation does not return missing values for the mask, why even use it in the first place?

This begs an important question for me:

Do we even need to call AEROBULK_INIT? I believe that most of the trouble we tried to fix in #11 was related to the range checking in AEROBULK_INIT (`ctl_stop` is only called in [`mod_aerobulk.f90`](https://github.com/brodeau/aerobulk/blob/115d6db4fde9a3c1943a46bf51867abb5ca903b4/src/mod_aerobulk.f90#L126-L153) and never in `mod_aerobulk_compute.f90`, which is the actual code we are wrapping). They only call `STOP` in two locations [here](https://github.com/brodeau/aerobulk/blob/115d6db4fde9a3c1943a46bf51867abb5ca903b4/src/mod_aerobulk_compute.f90#L108) and [here](https://github.com/brodeau/aerobulk/blob/115d6db4fde9a3c1943a46bf51867abb5ca903b4/src/mod_aerobulk_compute.f90#L174).

We already check the valid algo input in the xarray wrapper ([here](https://github.com/xgcm/aerobulk-python/blob/c066c58a7469ae4243380921fa30bc756086002b/source/aerobulk/flux.py#L11-L13)) and I think it would be reasonable to restrict out code to only accept one 'type' of humidity (specific humidity is the one that Aerobulk actually wants, and I think our models actually provide that).

Additionally I think that I am tending to think we should handle out-of-range-values on the numpy wrapper level, but we can discuss that in the meeting later today.

So in summary, I think it is worth trying to remove all calls to AEROBULK_INIT and just call AEROBULK_COMPUTE with values that we have cleaned on the numpy level?
